src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -197,7 +186,7 @@ public static Command startReceive(Conveyor conveyor) {
             new InstantCommand(() -> conveyor.frontMotor.set(ConveyorCal.RECEIVE_SPEED), conveyor),
             new InstantCommand(() -> conveyor.backMotor.set(ConveyorCal.RECEIVE_SPEED)),
             new InstantCommand(() -> conveyor.currentNotePosition = ConveyorPosition.PARTIAL_NOTE),
-            new WaitUntilCommand(() -> conveyor.beamBreakSensorOne.isPressed()))
+            new WaitUntilCommand(() -> !conveyor.frontConveyorBeamBreakSensor.get()))
```

Comment:
Can we just change this to back now..

Reply:
Done. Do we still need the slow speed logic? Or can we remove that now that we're using the two different sensors?

Reply:
Almost certainly still need to go slow. We'll test how slow. Might be fairly fast

Reply:
Well, you'll test. Tomorrow